wordpress-mobile/WordPress-Android
# Track Reader Discover Content Presented

Fixes #13609
iOS PR: [WordPress-iOS#15183](https://github.com/wordpress-mobile/WordPress-iOS/pull/15183)

This PR tracks if:

1) The API response has arrived.
2) The user is still waiting for the content/error to be presented (aka didn't left the `Discover` screen).

The event fired is intended to be used in a funnel, to check how many users are leaving the `Discover` screen due to the API taking too long to respond.

### To test

Launch the app:

1) Open `Discover` screen.
1) Wait for the content/error to be presented.
1) Check that `reader_discover_content_presented` is fired.

Launch the app again (to clear cache):

1) Open `Discover` screen.
1) Immediately switch to another section (like the `Following` screen).
1) Check that `reader_discover_content_presented` is not fired.

PR submission checklist:
